source/appendices/e-audit-requirements-and-tests/02-audit-tests.rst

```diff
@@ -5,1045 +5,391 @@
-
-
-Audit Test A1-2: Total_Item_Requests and Unique_Item_Requests - 30-second filters
-.................................................................................
-
-The audit test consists of clicking links to a journal article item twice in succession (double-clicks). If the two clicks occur within a 30-second time-span, only the second Total_Item_Requests must be recorded. If the two clicks occur with more than 30 seconds between, then 2 Total_Item_Requests must be counted. In both cases, only 1 Unique_Item_Request will be reported.
-
-The auditor must carry out a total of 30 tests, and each test will consist of 2 requests. There are 2 types of tests that must be carried out:
-
-* “Inside” tests (Two requests are made to the same journal article item, and the second request is made within 30 seconds of the first).
-* “Outside” tests (Two requests are made to the same journal article item, and the second request is made more than 30 seconds after the first).
-
-The auditor must carry out 15 **inside** tests.
-
-This must result in:
-
-* 15 Total_Item_Requests and 15 Unique_Item_Requests in the IR_A1 Standard View.
-
-The auditor must carry out 15 **outside** tests.
-
-This must result in:
-
-* 30 Total_Item_Requests and 15 Unique_Item_Requests  in the IR_A1 Standard View.
-
-
-E.2.4.3 Standard View: IR_M1
-''''''''''''''''''''''''''''
+E.2.8 Audit Tests for Denials
+"""""""""""""""""""""""""""""
 
-Reports on multimedia requests at the item level.
+Report providers operating platforms where turnaways or denials are in operation MUST be subject to audit tests for denials. For report providers operating multiple platforms, the audit scope as defined in :numref:`audit` MUST include platforms where turnaways or denials are in operation. Where either Limit_Exceeded or No_License denials do not apply to a report provider, auditors MUST note this in the audit report. This does not require an exemption from the COUNTER Project Director.
 
-An audit of this Standard View requires the following:
+These audit tests apply to denial metrics across all COUNTER Reports and should represent the scope of the platform under audit. That is, where a platform is made up of a mixture of content with Data_Types Article, Multimedia and Patent, the auditor SHOULD represent each of those Data_Types proportionately in the audit test.
 
-* The auditor must have access to all multimedia content available by the content provider.
-* The auditor must allow at least 31 seconds between each request, unless otherwise specified.
-* Audit tests M1-1 and M1-2 must take place in separate accounts so that each audit test can be separately reported.
-* For each applicable audit test to achieve an audit pass, all metrics reported by the content provider in IR_M1 Standard View from the auditor’s test account must be within a -8% and +3% reliability window of the same audit metrics on the auditor’s report.
 
+E.2.8.1 Limit_Exceeded
```

Comment:
The wording in the section seems to suggest that Limit_Exceeded only applies to databases, but this isn't the case.